exchange/utils.go

```diff
@@ -227,7 +229,10 @@ func getAuctionBidderRequests(auctionRequest AuctionRequest,
 		reqCopy := *req.BidRequest
 		reqCopy.Imp = imps
 
-		sChainWriter.Write(&reqCopy, bidder)
+		err := sChainWriter.Write(&reqCopy, bidder, hostSChainNode)
+		if err != nil {
+			return nil, []error{err}
+		}
```

Comment:
I'm pretty sure we intentionally left out this error handling because we didn't want invalid schains killing the auction. I'll need to check on that.